src/OpenSage.Game/Logic/Object/Draw/W3dModelDraw.cs

```diff
@@ -166,7 +175,7 @@ private void SetActiveAnimationState(AnimationState animationState)
             }
         }
 
-        public override void UpdateConditionState(BitArray<ModelConditionFlag> flags)
+        public override void UpdateConditionState(BitArray<ModelConditionFlag> flags, Random random = null)
```

Comment:
Why have the default value for `random`?

Reply:
Same scenario as above -> this calls later FloatRange.GetValue(Random)